### PR TITLE
feat(google maps): add min and max zoom prop

### PR DESCRIPTION
**Pull request by tadjohnston**
affects: @rentpath/react-ui-core

- adding props for minZoom and maxZoom
- adding default minZoom of 5

### CODE REVIEW

File: packages/react-ui-core/src/Gmap/Gmap.js [hunk 1]
```diff
@@ -124,13 +128,17 @@ export class Gmap extends PureComponent {
...
+      maxZoom,
       clickableIcons,
       stylingFunction,
     } = this.props
 
     this.map = new window.google.maps.Map(this.googleMap.current, {
       zoom,
       clickableIcons,
+      minZoom,
+      maxZoom,
```
Comment (barelyfitz) [inline packages/react-ui-core/src/Gmap/Gmap.js]: Add unit tests:

* minZoom prop is not set --> it uses the default minZoom
* minZoom prop is set --> it uses the provided minZoom
* maxZoom prop is not set --> it uses the default maxZoom
* maxZoom prop is set --> it uses the provided maxZoom

Comment (tadjohnston) [reply packages/react-ui-core/src/Gmap/Gmap.js]: Tests added.

File: packages/react-ui-core/src/Gmap/Gmap.js [hunk 2]
```diff
@@ -124,13 +128,17 @@ export class Gmap extends PureComponent {
...
       center: center || DEFAULT_CENTER,
```
[the remark this answers is not in the document]
Comment (tadjohnston) [reply packages/react-ui-core/src/Gmap/Gmap.js]: There was a reason we did this, I'm trying to think what it was.

[the remark this answers is not in the document]
Comment (tadjohnston) [reply packages/react-ui-core/src/Gmap/Gmap.js]: I can't think of a reason _not_ to change it. Added a test for it as well.

**Pull request merged**